tests/test_datagetter.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+from tests.object_mother import ObjectMother
+from tests.utils import OpenEventTestCase
+from tests.setup_database import Setup
+from tests.auth_helper import register, logout
+from open_event import current_app as app
+from open_event.models import db
+from open_event.forms.admin.track_form import TrackForm
+from open_event.helpers.data import DataManager
+
+
+class TestSessionForm(OpenEventTestCase):
+
+    def setUp(self):
+        self.app = Setup.create_app()
+        with app.test_request_context():
+            db.session.add(ObjectMother.get_event())
+            db.session.commit()
+
+    def _create_track(self):
+        form = TrackForm()
+        form.name.data = 'SomeTrack'
+        form.description.data = 'With Description'
+        form.track_image_url.data = 'https://example.com/image.gif'
+        DataManager().create_track(form, 1)
+
+    def _delete_track(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2539694755/issues/source?bid=3233864&fileBranchId=3285268#l28)